docs/about.md

```
@@ -17,6 +17,7 @@ title: About
   <a href="https://www.gnu.org/licenses/agpl-3.0">
     <img alt="License" src="https://img.shields.io/github/license/twangodev/uw-coursemap" />
   </a>
+  <a title="Crowdin" target="_blank" href="https://crowdin.uwcourses.com/"><img src="https://badges.crowdin.net/uw-coursemap/localized.svg"></a>
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)